libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/UrlUtils.kt

```diff
@@ -0,0 +1,167 @@
+/*
```

Comment:
@LukasPaczos Not sure if this is the right way to embed other sources in our codebase.

The previous implementation was URL encoding "ssml" voice instructions in a specific way (see _String.canonicalize_ method below) that `api.mapbox.com` accepts. Unfortunately, that encoding was done implicitly by a utility class in okhttp3 lib that's used by `com.mapbox.mapboxsdk:mapbox-sdk-services`.